examples/alt_key_builder.py

```diff
@@ -0,0 +1,293 @@
+"""alt_key_builder.py
```

Comment:
This file feels a little too verbose/dense for an example, but I'll merge it for now and look at it again in a later release.